csi.proto

```diff
@@ -945,6 +993,21 @@ message ControllerServiceCapability {
       // Indicates the SP supports the
       // ListVolumesResponse.entry.published_nodes field
       LIST_VOLUMES_PUBLISHED_NODES = 10;
+
+      // Indicates the SP supports the
+      // ListVolumesResponse.entry.volume_condition field
+      LIST_VOLUMES_VOLUME_CONDITION = 11 [(alpha_enum_value) = true];
+
+      // Indicates the SP supports the GetVolume RPC
+      GET_VOLUME = 12 [(alpha_enum_value) = true];
+
+      // Indicates the SP supports the
+      // GetVolumeResponse.volume_condition field
```

Comment:
what's the different between this and `LIST_VOLUMES_VOLUME_CONDITION` ? I think you either need to call this `GET_VOLUME_VOLUME_CONDITION`, or rename `LIST_VOLUMES_VOLUME_CONDITION` to `VOLUME_CONDITION` and use that for both GetVolume and ListVolumes

Reply:
Renamed to GET_VOLUME_VOLUME_CONDITION.